.github/workflows/library.yml

```diff
@@ -534,7 +534,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu", "macos", "windows"]
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8] #TODO: Does this need updating?
```

Comment:
Does this need updating?

Reply:
It's actually a "dead bit of code" not currently in use... Left over from an attempt to automate the release process.

Reply:
Likely still worth keeping around if we ever do want to automate the release process...

Reply:
but if we keep it around, shouldn't we update it to the versions of python being supported?
We at least support 3.9, we are looking at supporting 3.10, and without double checking (sorry), I thought we don't support 3.6 and are about to stop supporting 3.7...

Reply:
I edited them as per @sjswerdloff's points.